VultisigApp/VultisigApp/View Models/ImportVaultViewModel.swift

```diff
@@ -72,11 +72,11 @@ class ImportVaultViewModel: ObservableObject {
         }
         
         let decoder = JSONDecoder()
-        
         do {
             let backupVault = try decoder.decode(BackupVault.self,
-                                           from: vaultData)
+                                                 from: vaultData)
             // if version get updated , then we can process the migration here
+            ensureVaultUnique(backupVault: backupVault.vault,vaults:vaults)
             modelContext.insert(backupVault.vault)
```

Comment:
It seems it is inserting anyway, but showing the alert, no?

Reply:
fixed